roofit/roofitcore/inc/RooMinimizer.h

```diff
@@ -162,6 +163,9 @@ class RooMinimizer : public TObject {
 private:
    friend class RooAbsMinimizerFcn;
    friend class RooMinimizerFcn;
+#ifdef ROOFIT_MULTIPROCESS
+   friend class RooFit::TestStatistics::LikelihoodGradientJob;
+#endif // ROOFIT_MULTIPROCESS
```

Comment:
```suggestion
   friend class RooFit::TestStatistics::LikelihoodGradientJob;
```
Same as the other comment.